corgi/core/models.py

```diff
@@ -909,12 +909,13 @@ def is_srpm(self):
         return self.type == Component.Type.RPM and self.arch == "src"
 
     def get_nvr(self) -> str:
-        return f"{self.name}-{self.version}-{self.release}"
+        return f"{self.name}-{self.version}{f'-{self.release}' if self.release else ''}"
 
     def get_nevra(self) -> str:
         return (
             f"{self.name}{f':{self.epoch}' if self.epoch else ''}"
-            f"-{self.version}-{self.release}.{self.arch}"
+            f"-{self.version}{f'-{self.release}' if self.release else ''}"
+            f"{f'.{self.arch}' if self.arch else ''}"
```

Comment:
+1 to Martin's comment above. Maybe this is a little more readable:
```suggestion
        epoch = f":{self.epoch}" if self.epoch else ""
        release = f"-{self.release}" if self.release else ""
        arch = f".{self.arch}" if self.arch else ""

        return (
            f"{self.name}{epoch}"
            f"-{self.version}{release}"
            f"{arch}"
)
```

Reply:
Ok, I removed nested f-stings.